bugbug/models/defect.py

```diff
@@ -243,7 +243,7 @@ def get_bugbug_labels(self, kind="bug"):
         return {bug_id: label for bug_id, label in classes.items() if bug_id in bug_ids}
 
     def get_labels(self):
-        return self.get_bugbug_labels("bug")
+        return (self.get_bugbug_labels("bug"), {0, 1})
```

Comment:
Nit: the 0 and 1 can be in a list instead of a set

Reply:
Since you are transforming to a list in model.py, you might as well have a list directly here (as you don't need the set to eliminate duplicate values like in other cases)

Reply:
(Similarly for the other models where you have a predefined list of classes)